plugins/experimental/access_control/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+#######################
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+#  agreements.  See the NOTICE file distributed with this work for additional information regarding
+#  copyright ownership.  The ASF licenses this file to you under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with the License.  You may obtain
+#  a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing permissions and limitations under
+#  the License.
+#
+#######################
+
+project(access_control)
+
+add_atsplugin(access_control
+    access_control.cc
+    common.cc
+    config.cc
+    headers.cc
+    pattern.cc
+    plugin.cc
+    utils.cc
+)
+
+target_include_directories(access_control PRIVATE "${PROJECT_SOURCE_DIR}/include")
+
+target_link_libraries(access_control
+    PRIVATE
+        "${PCRE_LIBRARY}"
+        "${OPENSSL_SSL_LIBRARY}"
+)
```

Comment:
Needs both OpenSSL libraries (`OPENSSL_LIBRARIES`) and `OPENSSL_INCLUDE_DIRS` and `PCRE_INCLUDE_DIRS` need to be added to the target include directories.